Updated version of 0006-linux-user-signal.c-define-__SIGRTMIN-MAX-for-non-GN.patch for qemu-3.0.0
from alpine, original author Natanael Copa.
--- linux-user/signal.c	2019-08-22 19:46:40.369463327 +0200
+++ linux-user/signal.c	2019-08-22 19:47:25.176898649 +0200
@@ -25,6 +25,13 @@
 #include "trace.h"
 #include "signal-common.h"
 
+#ifndef __SIGRTMIN
+#define __SIGRTMIN 32
+#endif
+#ifndef __SIGRTMAX
+#define __SIGRTMAX (NSIG-1)
+#endif
+
 static struct target_sigaction sigact_table[TARGET_NSIG];
 
 static void host_signal_handler(int host_signum, siginfo_t *info,
